cmd/release-controller/sync_tags.go

```diff
@@ -96,10 +96,23 @@ func (c *Controller) replaceReleaseTagWithNext(release *Release, tag *imagev1.Ta
 
 func (c *Controller) removeReleaseTags(release *Release, removeTags []*imagev1.TagReference) error {
 	for _, tag := range removeTags {
-		// use delete imagestreamtag so that status tags are removed as well
-		klog.V(2).Infof("Removing release tag %s", tag.Name)
-		if err := c.imageClient.ImageStreamTags(release.Target.Namespace).Delete(context.TODO(), fmt.Sprintf("%s:%s", release.Target.Name, tag.Name), metav1.DeleteOptions{}); err != nil {
-			if !errors.IsNotFound(err) {
+		// indicate by annotation that the isTag should be deleted
```

Comment:
This behavior should be controlled by a flag (if block, old behavior, new behavior, `--soft-delete-release-tags`).

Reply:
Added the flag with `false` as it default value.